addons/website/views/website_templates.xml

```diff
@@ -2589,6 +2589,61 @@ Sitemap: <t t-esc="url_root"/>sitemap.xml
     </xpath>
 </template>
 
+<template id="website.step_wizard" name="Step Wizard">
+    <!-- REQUIRED WHEN CALLING -->
+    <!-- o_wizard_activestep = int active step (from 0) -->
+    <!-- o_wizard_steps_text = array of step text -->
+    <!-- o_wizard_steps_href = array of step href -->
+    <!-- o_wizard_back_default = <- back href if @first step of the stepper -->
+
+    <!-- OPTIONAL -->
+    <!-- o_wizard_back_custom = <- back href if @any step of the stepper -->
+
+    <t t-set="o_wizard_steps_in_deg" t-value="(o_wizard_activestep + 1) / len(o_wizard_steps_text) * 360"/>
+
+    <div class="o_wizard d-flex flex-wrap justify-content-end justify-content-sm-start my-3 my-sm-4">
+        <a class="align-self-center me-auto" t-attf-href="{{ o_wizard_back_custom if o_wizard_back_custom else o_wizard_steps_href[o_wizard_activestep - 1] if o_wizard_activestep &gt; 0 else o_wizard_back_default if o_wizard_back_default else '#'}}"><i class="fa fa-long-arrow-left me-2"/>Back</a>
```

Comment:
Could you move inner content in a new line?
It would improve code readability... also it would limit diffs if case we'll need to change just the content or the condition in the future
```suggestion
        <a class="align-self-center me-auto" t-attf-href="{{ o_wizard_back_custom if o_wizard_back_custom else o_wizard_steps_href[o_wizard_activestep - 1] if o_wizard_activestep &gt; 0 else o_wizard_back_default if o_wizard_back_default else '#'}}">
        <i class="fa fa-long-arrow-left me-2"/>Back</a>
```